src/supertux/menu/joystick_menu.cpp

```diff
@@ -187,31 +187,31 @@ JoystickMenu::refresh_menu_item(Controller::Control id)
     switch (hat_dir)
     {
       case SDL_HAT_UP:
-        name = "Hat Up";
+        name = _("Hat Up");
         break;
 
       case SDL_HAT_DOWN:
-        name = "Hat Down";
+        name = _("Hat Down");
         break;
 
       case SDL_HAT_LEFT:
-        name = "Hat Left";
+        name = _("Hat Left");
         break;
 
       case SDL_HAT_RIGHT:
-        name = "Hat Right";
+        name = _("Hat Right");
         break;
 
       default:
-        name = "Unknown hat_dir";
+        name = _("Unknown hat_dir");
```

Comment:
There are other hat directions:
https://www.libsdl.org/tmp/docs-1.2/_s_d_l__joystick_8h.html
But on the other hand, SuperTux doesn't support them:
https://github.com/SuperTux/supertux/blob/ed17b8153a084a1d34845414a0bbc4bf7c669ff8/src/control/joystick_config.cpp#L203
So really this should be an error.